### PR TITLE
[13.0] [ADD] Account Fiscal Position Allowed Journal Purchase

### PR DESCRIPTION
**Pull request by ThomasBinsfeld**
Based on https://github.com/OCA/account-financial-tools/pull/937

### CODE REVIEW

**Review by chrisdec — APPROVED** [review]
Functional tests

**Review by chrisdec — APPROVED** [review]
Functional test

**Review by chrisdec — APPROVED** [review]
Functional test

**Review by sanrav — APPROVED** [review]
Functional test OK

File: .travis.yml [hunk 1]
```diff
@@ -37,6 +37,11 @@ env:
     - VERSION="13.0" TESTS="0" LINT_CHECK="0" MAKEPOT="0"
 
 install:
+  #  FIXME:
+  - git clone --branch 13.0-add_account_fiscal_position_allowed_journal_tbi
+    https://github.com/acsone/account-financial-tools.git PR937
+  - rm -rf account_fiscal_position_allowed_journal && cp -ar
+    PR937/account_fiscal_position_allowed_journal .
```
Comment (HviorForgeFlow) [inline .travis.yml]: Waiting dependencies, this should be removed, shouldn't it?

**Review by HviorForgeFlow — CHANGES_REQUESTED** [review]

**Comment by HviorForgeFlow** [comment]
@ThomasBinsfeld  dependency has been merged, can you drop oca_dependencies patch? Thanks!

**Comment by ThomasBinsfeld** [comment]
> @ThomasBinsfeld dependency has been merged, can you drop oca_dependencies patch? Thanks!

Done

**Review by HviorForgeFlow — APPROVED** [review]

**Comment by HviorForgeFlow** [comment]
@ThomasBinsfeld I think a manual rebase is needed

**Comment by dreispt** [comment]
Merges with #2277

**Pull request closed without merging**